content/SCALE/SCALETutorials/Storage/ManagePoolsScale.md

```diff
@@ -149,6 +148,10 @@ To manually recover capacity, simply replicate and rewrite the data to the exten
 
 You can use the [RAIDZ Extension Calculator]({{< relref "/References/ExtensionCalculator.md" >}}) to visualize potential lost headroom and capacity available to recover by rewriting existing data.
 
+Note: While this process can recover the actual lost capacity, reported capacity continues to rely on the old data-to-parity ratio.
+An expanded vdev can continue to report a lower than expected capacity, even after rewriting old data to the new parity ratio.
+This accounting inconsistency does not impact the actual available capacity of the vdev.
+
```

Comment:
same comment as above. should you make this a snippet and use a hint tip or info type for it?